### DISCUSSION TITLE
CudaText editor lexer for Dictu

### DISCUSSION THREAD

**Discussion by Alexey-T** [Jason2605/Dictu#563] [Show and tell]
we had the 'issue' in GH about this. it was removed somehow (by GH stuff). I repeat here.
CudaText editor now has the Dictu lexer.
to install it: menu item "Plugins / Addons Manager / Install", choose 'Dictu' item.

![Screenshot from 2022-11-15 00-01-38](https://user-images.githubusercontent.com/4154978/201764719-945c5733-1fd5-4ee5-a844-3736f6baa1b7.png)

you may post here about lexer issues.

![dictu](https://user-images.githubusercontent.com/4154978/201764527-27fa7661-903e-4d25-b94d-3e7bea9b5725.png)

**Comment by Jason2605** [comment]
Yeah unsure what happened to that issue, this looks nice though!
